src/hooks/semesters/use-semesters.hook.ts

```diff
@@ -7,7 +7,7 @@ import { useRecoilCallback } from 'recoil'
 
 type HandleRefresh = () => Promise<GetSemestersResDTO>
 export const useSemesters = () => {
-  const handleRefresh: HandleRefresh = useRecoilCallback(
+  const onSemesters: HandleRefresh = useRecoilCallback(
```

Comment:
onGetSemsters

Reply:
Handled in [5ddf33e](https://github.com/ajktown/wordnote/pull/158/commits/5ddf33e05136c5e01ce02ca05840754462e6215c)